CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 This changelog follows the format defined here: https://keepachangelog.com/en/1.0.0/
 
+## [1.6.0]
+
+A more detailed account of everything included is included in the dev->master PR for this release: https://github.com/firesim/firesim/pull/XXX
+
+### Added
+* PR #250 Add support for flow control via Ethernet pause frames.
+* Adds support for runtime-configurable MSHRs.
+* Support for custom FireSim Base AMI (instead of generic FPGA Developer AMI)
+
+### Changed
+* PR #261 Print out that post run hook is running, so it doesn't look like simulation termination is stuck.
+
+### Fixed
+* PR #275. Fix small root volume size for build instances. 
+    * Increased to 100GB
+    * Resolves #274
+    * Resolves #265
+* PR #280. Add support for a FIRESIM_RUNFARM_PREFIX env var to differentiate between runfarms for multiple clones of firesim. 
+    * Resolves #263
+* PR #286 Add support for arbitrary host frequency selection (by synthesizing a new PLL). 
```

Comment:
This feels like an "Added" but i'm easy

Reply:
My rule of thumb was that if it closes and issue, then it was a fix. But I don't mind moving things

Reply:
But I agree, so I moved them to "Added"